### DISCUSSION TITLE
关于轮播组件 Swipe 使用的一些问题

### DISCUSSION THREAD

**Discussion by SPGLP** [youzan/vant#12807] [Q&A] [answered]
本人和本人的同学按照 Vant 官网上的文档，调试多次，最后甚至直接复制粘贴了，但还是显示成如下这种情况，无法完成正常轮播。换成图片也是一样，求大佬帮帮孩子啊 T_T。

- 已经分别在两人的设备上都进行过调试了，显示效果是一样的
- 同学使用的是 Windows 10 系统，我使用的是 Windows 11
- 两人均使用新版的 Microsoft Edge 浏览器
- node.js 等都已配置完毕，版本为最新的LTS。

具体的效果是，网页加载完毕后所有元素都直接全部垂直显示，然后所有图片往左移动，页面变为空白。个人推测轮播效果已经启动，但不知为何轮播元素无法正常显示。

**所有代码都是直接复制官网文档的，本人没有添加任何其它的样式和script**

情况截图如下：
<img width="932" alt="report" src="https://github.com/youzan/vant/assets/67890540/f7539d4f-5133-4c33-810d-2d2fcf96ad89">

**Comment by inottn** [comment]
提供下复现链接或者代码仓库吧，可能是对应的样式没有引入

**Reply by SPGLP** [reply]
gitee仓库：

[https://gitee.com/chaoix/schoolwork-vant-test.git](https://gitee.com/chaoix/schoolwork-vant-test.git)

这是我们的学校作业T_T，我和同学都是刚学，不太会。仓库里的是原本打算实现的效果，在 src\components\pages\CommonRight2.vue 里面是 Swipe 的工作组件，是轮播显示图片的。我后来把它们换成官网代码，显示纯文本也是这样的效果，所以就直接把更改撤销了，没有推上去。

**Reply by inottn** [reply, marked as answer]
样式没有引入，在 main.js 里 import 'vant/lib/index.css';

**Reply by SPGLP** [reply]
成功了，谢谢大佬！ ヾ(≧▽≦*)o
我同学还在课本上划过的，但他忘了，我当时也没注意这点。 ＞﹏＜
